c++/include/orc/Type.hh

```diff
@@ -64,7 +64,8 @@ namespace orc {
      * Create a row batch for this type.
      */
     virtual ORC_UNIQUE_PTR<ColumnVectorBatch> createRowBatch(uint64_t size,
-                                                             MemoryPool& pool
+                                                             MemoryPool& pool,
+                                                             bool encoded = false
```

Comment:
rename into "dictionaryEncoded"?

Reply:
Same as above. I think we better leave it as encoded for future support like RLE lazy decoding.